ansible_collections/arista/avd/roles/eos_designs/templates/base/base.py

```diff
@@ -0,0 +1,148 @@
+from ansible_collections.arista.avd.plugins.module_utils.eos_designs import EosDesigns
+from functools import cached_property
+from ansible_collections.arista.avd.plugins.module_utils.utils import get
+
+
+class AVDStructConfig(EosDesigns):
+    def __init__(
```

Comment:
please use __init__ from base class. Notice it uses _hostvars, so you will have to change that reference in the other functions here.